src/Administration/Resources/app/administration/src/app/component/form/sw-colorpicker/sw-colorpicker.scss

```diff
@@ -162,21 +162,14 @@
             border: 1px solid $color-steam-cloud;
             background: #fff;
             -webkit-appearance: none;
-        }
-
-        &-slider-range::-webkit-slider-thumb:hover {
             cursor: pointer;
         }
```

Comment:
Please let this css selector still exist with its content and add an deprecation tag E.g. `/* @deprecated tag:v6.3.0 */`. Duplicate the css cursor prop and put it in the right selector as you did before.